default.py

```diff
@@ -47,6 +49,14 @@ def addon_log(string):
     msg = '%s: %s' % (LOGGING_PREFIX, string)
     xbmc.log(msg=msg, level=xbmc.LOGDEBUG)
 
+def ShowBusyDialog():
+    busydialog.create()
+
+def HideBusyDialog():
+    try:
+        busydialog.close()
+    except RuntimeError,e:
```

Comment:
Thanks for the explanation. The closing problem sounds familiar now; it's been awhile since I last touched the busy dialogs.

No need for the boolean state. I think the wrapper functions is a cleaner approach.

---Alex